cmd/argocd/commands/app.go

```diff
@@ -752,7 +751,7 @@ func groupLocalObjs(localObs []*unstructured.Unstructured, liveObjs []*unstructu
 	objByKey := make(map[kube.ResourceKey]*unstructured.Unstructured)
 	for i := range localObs {
 		obj := localObs[i]
-		if !(hook.IsHook(obj) || resource.Ignore(obj)) {
```

Comment:
I think we might start showing ignored helm hooks in UI/CLI because of this change. E.g. if chart has `helm.sh/hook-delete-timeout` argocd should ignore it and don't even show in app tree/sync status panel.

Reply:
interesting - I'll need to try this out to see